monster-cards/tests/unit/main.ts

```diff
@@ -0,0 +1,10 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import * as main from 'src/main';
```

Comment:
if you switch to `node` resolution you will need to change this to a relative path